AdminPanel/fuctionalities/gettingBlogsInTable.js

```diff
@@ -9,7 +9,7 @@ if (getAllArticles.length > 0) {
         <tr>
             <td>${item.blogTitle}</td>
             <td>${item.blogDate}</td>
-            <td>0</td>
+            <td>${item.comments.length}</td>
             <td>0</td>
```

Comment:
if you don't need this table data you can remove it instead of living it as default of 0